nixos/modules/services/home-automation/matter-server.nix

```
@@ -0,0 +1,123 @@
+{ lib
+, pkgs
+, config
+, ...
+}:
+
+with lib;
+
+let
+  cfg = config.services.matter-server;
+in
+
+{
+  meta.maintainers = with lib.maintainers; [ hexa ];
+
+  options.services.matter-server = with types; {
+    enable = mkEnableOption (lib.mdDoc "Matter-server");
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.python-matter-server;
+      defaultText = literalExpression "pkgs.python-matter-server";
+      description = "Package containing the matter-server executable.";
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 5580;
+      defaultText = "5580";
+      description = "Port to expose the matter-server service on.";
+    };
+
+    vendorId = mkOption {
+      type = types.ints.u16;
+      default = 4939; # Home Assistant
+      defaultText = "4939 (Home Assistant)";
+      description = "CSA Vendor ID to advertise to devices on the network.";
+    };
+
+    fabricId = mkOption {
+      type = types.int;
+      default = 1;
+      defaultText = "4939 (Home Assistant)";
```

Mismatch, but `defaultText` is redundant anyway.

```suggestion
      default = 1;
```

---

Done.